it/uk/gov/hmrc/fhdds/testsupport/TestConfigures.scala

```diff
@@ -14,7 +14,7 @@ import com.github.tomakehurst.wiremock.client.WireMock.{configureFor, reset, res
 import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
 import uk.gov.hmrc.fhdds.testsupport.verifiers.VerifierBuilder
 
-trait TestedApplication
+trait TestConfigures
```

Comment:
Should this be TestConfiguration rather than TestConfigures?

Reply:
have renamed the trait from TestConfigures to TestConfiguration